.github/workflows/coverage.yml

```diff
@@ -0,0 +1,47 @@
+name: coverage
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
Does it matter that the python version here is 3.7 while the one above is 3.6?

Reply:
ah! Nice, yeah, I can change the defaults here. I don't think it "matters" since we have a build matrix handle all three versions in `workflow.yml` but let's pin to the default we support.